doc/fuzzing.md

```diff
@@ -7,11 +7,16 @@ To quickly get started fuzzing Bitcoin Core using [libFuzzer](https://llvm.org/d
 ```sh
 $ git clone https://github.com/bitcoin/bitcoin
 $ cd bitcoin/
-$ ./autogen.sh
-$ CC=clang CXX=clang++ ./configure --enable-fuzz --with-sanitizers=address,fuzzer,undefined
+$ mkdir fuzz_build && cd fuzz_build
+$ cmake -S .. \
+   -DCMAKE_C_COMPILER="clang" \
+   -DCMAKE_CXX_COMPILER="clang++" \
+   -DCMAKE_BUILD_MODE=None \
```

Comment:
Should this be in here or can i just drop it?

Reply:
Let's wait for the discussion in https://github.com/hebasto/bitcoin/pull/82.